examples/base_ac/main.tf

```diff
@@ -39,7 +39,7 @@ resource "aws_key_pair" "deployer" {
 
 resource "local_file" "private_key" {
   content         = tls_private_key.key.private_key_pem
-  filename        = "../${var.name_prefix}-key-${random_string.suffix.result}.pem"
+  filename        = "./${var.name_prefix}-key-${random_string.suffix.result}.pem"

```

Comment:
@willguibr no concerns here. just curious what was the driver to bump this file right up to the specific examples dir rather than /examples itself?

Reply:
@jmolnar-zscaler When you deploy, the VM, then to authenticate to it after, you have to go out of the i.e base_ac folder in order to use the key. I found it easier to store all generated artifacts directly in the folder you're already executing the module from. Was there a design issue with that?